server/cmwell-sparql-agent/src/main/scala/cmwell/tools/data/sparql/SparqlProcessorManager.scala

```diff
@@ -227,105 +234,84 @@ class SparqlProcessorManager (settings: SparqlProcessorManagerSettings) extends
     }
   }
 
-/*
+
   /**
     * Generates data for tables in cm-well monitor page
     */
-  def stringifyActiveJobs(activeJobs: Map[String, JobStatus], currentNonActiveConfigs: Configs): Future[Iterable[Table]] = {
+  def stringifyActiveJobs(jobs: Jobs): Future[Iterable[Table]] = {
     implicit val timeout = Timeout(1.minute)
 
-    def generateNonActiveTables(currentNonActiveConfigs: Configs) = currentNonActiveConfigs.map { case (path, config) =>
-      val sensorNames = config.sensors.map(_.name)
-      val title = Seq(s"""<span style="color:red"> **Non-Active** </span> ${path.utf8String}""")
-      val header = Seq("sensor", "point-in-time")
+    def generateNonActiveTables(jobs: Jobs) = jobs.collect { case (path, jobStatus@(_: JobPaused | _:JobFailed)) =>
```

Comment:
JobPaused and JobFailed are unified.
JobFailed has another field that exposes the failure reason (exception). It should be shown in the dashboard.